scripts/qa_crowbarsetup.sh

```diff
@@ -2974,6 +2974,10 @@ function custom_configuration()
                 local cinder_volume
                 # fetch one of the compute nodes as cinder_volume
                 cinder_volume=`printf "%s\n" $unclustered_nodes | tail -n 1`
+                # make sure we do not use SP1 node for cinder
+                if [ -n "$deployceph" -a iscloudver 7plus ] ; then
```

Comment:
no hard opinion, but IIRC we were moving from `[` to `[[`:

`[[ -n "$deployceph" && iscloudver 7plus ]]  && cinder_volume=$sles12plusnode`

Also IIUC ... is this doing the opposite? If deployceph and is a SOC7, cinder_volume is a node with target=suse-12.1 [1]

[1] https://github.com/jsuchome/automation/blob/master/scripts/qa_crowbarsetup.sh#L759